tests/kubernetes/test_pod_generator.py

```diff
@@ -658,11 +658,12 @@ def test_deserialize_model_file(self):
     def test_pod_name_confirm_to_max_length(self, _, pod_id):
         name = PodGenerator.make_unique_pod_id(pod_id)
         assert len(name) <= 253
-        parts = name.split(".")
-        if len(pod_id) <= 63:
+        parts = name.split("-")
+        if len(pod_id) <= 63 - 33:
```

Comment:
Maybe add a comment to explain why 63 and 33 are significant?

Reply:
I added a few notes explaining where these numbers came from.